File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/SearchTournamentPresenter.java
package bobrchess.of.by.belaruschess.presenter;

import bobrchess.of.by.belaruschess.view.activity.impl.SearchTournamentActivity;

/**
 * Created by Igor on 12.04.2018.
 */

public interface SearchTournamentPresenter extends BasePresenter{
    void loadTournaments();

    void loadTournaments(Integer count);

    void searchTournaments();

    void attachView(SearchTournamentActivity activity);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/SearchTournamentPresenter.kt
package bobrchess.of.by.belaruschess.presenter

import bobrchess.of.by.belaruschess.view.activity.SearchTournamentContractView

interface SearchTournamentPresenter : BasePresenter {
    fun loadTournaments()
    fun loadTournaments(count: Int?)
    fun searchTournaments(text: String)
    fun attachView(searchTournamentContractView: SearchTournamentContractView)
}
